build-advisories/src/Roave/SecurityAdvisories/Version.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Roave\SecurityAdvisories;
+
+/**
+ * A simple version, such as 1.0 or 1.0.0.0 or 2.0.1.3.2
+ */
+final class Version
+{
+    const VALIDITY_MATCHER = '^(\d+.)*\d+$';
```

Comment:
This is broken: the dot needs escaping.

and you should use a non-capturing group and possessive quantifiers to optimize the regex